tox.ini

```diff
@@ -19,23 +20,9 @@ basepython =
     py35: python3.5
 
 deps =
-    django-modelcluster>=1.1b1
-    django-taggit>=0.17.5
-    django-treebeard>=3.0,<5.0
     django-sendfile==0.3.6
```

Comment:
Would moving `django-sendfile`, `Embedly`, and `jinja2` deps in to the `[testing]` requirements work? Then all the requirements would live in the `setup.py` `elasticsearch` is an optional dependency that is required for testing that already lives in `setup.py` `[testing]`, so it seems that it might work.